src/plugins/email.py

```diff
@@ -8,6 +8,7 @@
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 from email.utils import formatdate
+from email.utils import formataddr
```

Comment:
imports from same lib can be combined to single line
`from email.utils import fromatdate, formataddr`

Reply:
Agreed, modifying it